ztp/kube-compare-reference/metadata.yaml

```diff
@@ -40,20 +61,29 @@ Parts:
           - path: required/machine-config/sriov-related-kernel-arguments/07-sriov-related-kernel-args-worker.yaml
           - path: required/machine-config/crun/enable-crun-master.yaml
           - path: required/machine-config/crun/enable-crun-worker.yaml
+          - path: required/machine-config/kdump/06-kdump-master.yaml
+          - path: required/machine-config/kdump/06-kdump-worker.yaml
+  - name: required-namespace
+    Components:
+      - name: namespace
+        type: Required
+        requiredTemplates:
+          - path: required/namespace/09-openshift-marketplace-ns.yaml
   - name: required-node-tuning-operator
     Components:
       - name: node-tuning-operator
         type: Required
         requiredTemplates:
           - path: required/node-tuning-operator/PerformanceProfile.yaml
-#          - path: required/node-tuning-operator/TunedPerformancePatch.yaml
+          - path: required/node-tuning-operator/TunedPerformancePatch.yaml
   - name: required-ptp-operator
     Components:
       - name: ptp-operator
         type: Required
         requiredTemplates:
           - path: required/ptp-operator/PtpConfigBoundary.yaml # TODO: the INI files embedded in PTPConfig are not handled. CNF-13528
-#          - path: required/ptp-operator/PtpConfigGmWpc.yaml # TODO: Cluster may not have all the PTPconfigs listed in metadata.yaml and currently we do not have the ability to suitable one to diff
+#          - path: required/ptp-operator/PtpConfigGmWpc.yaml # TODO: Cluster may not have all the PTPconfigs listed in metadata.yaml and currently we do not have the ability choose a suitable one dynamically. CNF-13832
+#          - path: required/ptp-operator/PtpOperatorConfig.yaml
```

Comment:
Should this be commented out? This is the operator config not a ptpconfig?

Reply:
Moved PtpOperatorConfig to optional and commented out PtpOperatorConfigForEvent...with comment saying one of them but not both